stage1/stdlibdirs.mk

```diff
@@ -8,8 +8,12 @@
 
 ifeq ($(SLD_INCLUDED),)
 
+SLD_SPACE :=
+SLD_SPACE +=
+
 SLD_INCLUDED := x
 SLD_LOCATIONS := $(shell ld --verbose | grep SEARCH_DIR | sed -e 's/SEARCH_DIR("=*\([^"]*\)");*/\1/g')
+SLD_LOCATIONS := $(foreach l,$(SLD_LOCATIONS),$l$(SLD_SPACE)$l/systemd)
```

Comment:
`SLD_LOCATIONS += $(foreach l,$(SLD_LOCATIONS),$l/systemd)` should be fine.

But anyway, something like `/usr/lib64/systemd` rather shouldn't be considered to be a lib dir that is used by a dynamic loader, right?